task-tracker-console-application/src/main/java/com/isa/menuafterloggingin/MenuAfterLoggingIn.java

```diff
@@ -0,0 +1,47 @@
+package com.isa.menuafterloggingin;
```

Comment:
trochę skróciłbym tę nazwę pakietu

Reply:
albo nawet te oba menu zrobiłbym w 1 pakiecie `menu`, ale też zależy jaki plan na to macie, czy pozostałe opcje będziecie pchać do tego samego pakietu co menu czy osobno, ja proponuje osobno; więc wtedy takie menu może być jednym pakietem